exercises/wordy/canonical-data.json

```
@@ -15,6 +15,24 @@
       },
       "expected": 5
     },
+    {
+      "uuid": "18983214-1dfc-4ebd-ac77-c110dde699ce",
+      "description": "just a zero",
+      "property": "answer",
+      "input": {
+        "question": "What is 0?"
+      },
+      "expected": 0
+    },
+    {
+      "uuid": "607c08ee-2241-4288-916d-dae5455c87e6",
+      "description": "just a negative value",
```

Consistency: the word "number" is already used in the file
```suggestion
      "description": "just a negative number",
```